test/cases/000_config/010_kmod/kmod.yml

```diff
@@ -13,5 +13,13 @@ onboot:
      - /lib/modules:/lib/modules
     capabilities:
      - all
+services:
```

Comment:
thats very strange, and I thought we were already using it in onboot elsewhere. Just tested it and it works for me. Are you sure you are using the latest `moby`?